docs/developer_README.md

```diff
@@ -0,0 +1,55 @@
+# LlamaBot: A Pythonic bot interface to LLMs
+
+This is a `developer` README for LlamaBot.
+
+LlamaBot implements a Pythonic interface to LLMs,
+making it much easier to experiment with LLMs in a Jupyter notebook
+and build Python apps that utilize LLMs.
+All models supported by [LiteLLM](https://github.com/BerriAI/litellm) are supported by LlamaBot.
+
+
+## Setting up your developer environment
+
+Make sure that you have followed the [README.md](../README.md) before jumping on the next steps.
+
+## Creating a Conda virtual environment
+
+It is recommended to have [Conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) available on your system.
+
+```bash
+conda env create -f environment.yml
+```
+
+After the conda environment creation, you may need to do run the following commands.
+
+```bash
+conda activate llamabot 
+```
+
+The following command will install all your pip dependencies
+```bash
+pip install -e .
+```
+
+## Configuring your downloaded model
+
+If you are using a local model, you may need to update the `DEFAULT_LANGUAGE_MODEL` to your local model path
+at [config.py](../llamabot/config.py).
```

Comment:
Thanks for bringing up this point on configuration, @anujsinha3!

While the spirit is correct, I think I would avoid directly editing the source file, as the intended way for configuration to be done is to use:

```bash
llamabot configure default-model --model-name "ollama/mistral"
```

If the edit happens in the source `config.py` file, then it may be committed to source, and that will become the default configuration on new releases. As a default chosen by us, it should be something both accessible but also stable over time, so it's safer to avoid editing `config.py` unless there is a good reason for a change.

Reply:
Understood, Thanks! Updated.